components/collapsible/collapsible.test.js

```diff
@@ -5,54 +5,53 @@ import sinon from 'sinon';
 import axe from 'axe-core';
 import configA11y from '../../storybook/scripts/storybook-a11y-test.config';
 
-const content = '<div data-qa="content-element"> Test Text </div>';
+const contentOnExpanded = '<div data-qa="content-on-expanded-element"> Expanded Test Text </div>';
+const contentOnCollapsed = '<div data-qa="content-on-collapsed-element"> Collapsed Test Text </div>';
+const anchor = '<button data-qa="anchor-element">click me</button>';
+
 const baseProps = {
   anchorText: 'anchor text',
 };
 
 describe('Dialtone vue Collapsible Component Tests', function () {
   // Wrappers
   let wrapper;
-  let contentElement;
+  let contentOnExpandedElement;
+  let contentOnCollapsedElement;
   let contentWrapperElement;
   let anchorElement;
   let anchorSlotElement;
-  let slots = { content };
 
   // Environment
   const attrs = {
     css: false, // Important attr to let test-utils fire the (after-enter and after-leave) events correctly
   };
-  let props = baseProps;
-
-  const _clearChildWrappers = () => {
-    contentElement = undefined;
-    contentWrapperElement = undefined;
-    anchorElement = undefined;
-    anchorSlotElement = undefined;
-    slots = { content };
-  };
 
-  const _setChildWrappers = () => {
+  // Environment
+  const defaultSlots = { contentOnExpanded, contentOnCollapsed };
+  const defaultPropsData = baseProps;
+
+  const findAllElements = () => {
     anchorElement = wrapper.find('[data-qa="dt-button"]');
     anchorSlotElement = wrapper.find('[data-qa="anchor-element"]');
-    contentElement = wrapper.find('[data-qa="content-element"]');
-    contentWrapperElement = wrapper.getComponent('.d-dt-collapsible__content');
+    contentOnExpandedElement = wrapper.find('[data-qa="content-on-expanded-element"]');
+    contentOnCollapsedElement = wrapper.find('[data-qa="content-on-collapsed-element"]');
+    contentWrapperElement = wrapper.find('[data-qa="contentWrapper"]');
   };
 
-  const _mountWrapper = () => {
+  const mountWrapper = ({ propsData = defaultPropsData, slots = defaultSlots } = {}) => {
     wrapper = mount(DtCollapsible, {
-      props,
+      props: propsData,
       slots,
       attrs,
       global: {
         stubs: {
-          transition: false,
+          transition: true,
```

Comment:
@braddialpad I had to update the transition stub to `true` here, otherwise the transition element won't update its children. It's already stubbed by default in Vue 2, so I'm not sure why we set it to `false` here.

Reply:
There's a lot of weird stuff with transitions and testing in vue 3 and vue-test-utils keeps changing how it works so guessing that was leftover from something to do with that. Fine to change if it fixes the problem.